### ISSUE TITLE
Dynamic Life List: Sorting obs by "Date Added" actually sorts of "Date Observed"

### ISSUE THREAD

**Issue by tiwane** [inaturalist/inaturalist#3110]
Go to https://www.inaturalist.org/lifelists/tiwane?view=tree&details_view=observations&tree_mode=full_taxonomy&taxon_id=1 which should show all my animal observations sorted by "Date Added", oldest to newest. However, the first observation is [this one](https://www.inaturalist.org/observations/58517), which was added in 2012. The first animal observation I *added* is [this one](https://www.inaturalist.org/observations/667), which appears third. 

We should either change the label to "Date observed" or actually sort by "Date added" - I'd prefer the latter since the default is to sort by "Date added", newest to oldest.

**Comment by fredunit** [comment]
Can I claim this problem? and do you mean change the capitalized Added to added?

**Comment by kueda** [comment]
@fredunit, go for it. I'd just use whatever version of "date added" we already have translated. Holler if you have trouble getting a working dev env.

**Comment by fredunit** [comment]
So, this mean the function is not correct, and I need to review the logic behind and change it, am I right?

**Comment by kueda** [comment]
The changes in https://github.com/inaturalist/inaturalist/pull/3111/files don't really change the "logic" of anything, they just change the capitalization of three strings. The problem described in this issue is that the sort order on Tony's life list is not bey "date added" as the interface claims, so the cause of the incorrect sorting needs to be investigated. Maybe that means the API request needs a different parameter, maybe that means the API is broken somehow.

**Comment by fredunit** [comment]
So, which @tiwane is the image first submitted should be showed at first one, because the oldest to newest should be base on the date submitted, am I correct_
![problem](https://user-images.githubusercontent.com/52747821/125159439-ea350d00-e1a9-11eb-8d12-a0c3209d7973.jpg)

**Comment by kueda** [comment]
Yes, if the sort order says "Date Added" then the results should be sorted in that order. The important result to achieve is that the UI text should describe what is actually happening.

**Comment by elysiumplain** [comment]
Very old issue - looking at UI and ref https://github.com/adam-rankn/inaturalist/commit/9c6cb743d528783d2793e2448263ba307b8093bb it seems resolved. Want to close, @tiwane ?
